keys/cryptostore/holder.go

```diff
@@ -43,12 +43,20 @@ func (s Manager) Create(name, passphrase, algo string) (keys.Info, string, error
 	if err != nil {
 		return keys.Info{}, "", err
 	}
-	key := gen.Generate()
+
+	// 128-bits the the all the randomness we can make use of
+	secret := crypto.CRandBytes(16)
+	key := gen.Generate(secret)
 	err = s.es.Put(name, passphrase, key)
 	if err != nil {
 		return keys.Info{}, "", err
 	}
-	seed, err := s.codec.BytesToWords(key.Bytes())
+
+	// TODO: clean up type/kind handling with go-data
+	typ := key.Bytes()[0]
```

Comment:
Let's add a comment here to state that we'll be using secret to carry along the type